.env.example

```diff
@@ -11,3 +11,9 @@ SOCIALHOME_DOMAIN=127.0.0.1:8000
 SOCIALHOME_HTTPS=False
 
 MOCHA_TESTS=True
+
+# Set these variables for Docker Postfix (be sure the DATABASE_URL matches)
```

Comment:
`Postfix` -> `PostgreSQL`.

I'm also wondering why do we need these here at all? Why would anyone want to change these for a development database? IMHO, they should just be set in `docker-compose.yml`. If someone *wants* to change these, they can change that file - it's a local copy anyway not part of the repo. If possible, it would be nice to not have extra optional stuff in `.env.example` - it already has a few unnecessary things that should be cleaned out.

Reply:
This is a good question. Here's my reasoning:

The .env file is also used by docker-compose for its environment variables. So this means we only have one source of truth. But it's easily moved to the docker-compose file.